curriculum/js1/9.js

```diff
@@ -4,8 +4,11 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (num,i=7) => {
+  if((num+i)%7===0){
+    return num+i;
+  }
+  return solution(num, i-1);
```

Comment:
input 6 should return 7. I don't think this solution works.

Reply:
solutions works song, 6 returns 7. 
test input (6,7,15,22)
output(7,14,21,28)